src/main/java/com/synopsys/integration/alert/workflow/processor/NotificationReceiver.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.workflow.processor;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.google.gson.Gson;
+import com.synopsys.integration.alert.common.enumeration.FrequencyType;
+import com.synopsys.integration.alert.common.event.AlertEventListener;
+import com.synopsys.integration.alert.common.event.DistributionEvent;
+import com.synopsys.integration.alert.common.event.EventManager;
+import com.synopsys.integration.alert.common.event.NotificationEvent;
+import com.synopsys.integration.alert.common.persistence.accessor.NotificationManager;
+import com.synopsys.integration.alert.common.rest.model.AlertNotificationWrapper;
+import com.synopsys.integration.alert.common.workflow.MessageReceiver;
+
+@Component(value = NotificationReceiver.COMPONENT_NAME)
+public class NotificationReceiver extends MessageReceiver<NotificationEvent> implements AlertEventListener {
+    public static final String COMPONENT_NAME = "notification_receiver";
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private final NotificationManager notificationManager;
+    private final NotificationProcessor notificationProcessor;
+    private final EventManager eventManager;
+
+    @Autowired
+    public NotificationReceiver(final Gson gson, final NotificationManager notificationManager, final NotificationProcessor notificationProcessor, final EventManager eventManager) {
+        super(gson, NotificationEvent.class);
+        this.notificationManager = notificationManager;
+        this.notificationProcessor = notificationProcessor;
+        this.eventManager = eventManager;
+    }
+
+    @Override
+    public void handleEvent(final NotificationEvent event) {
+        if (event.getDestination().equals(NotificationEvent.NOTIFICATION_EVENT_TYPE)) {
```

Comment:
I prefer seeing the constant call `.equals()` so we don't have to worry about NPEs.
```java
NotificationEvent.NOTIFICATION_EVENT_TYPE.equals(event.getDestination())
```